### PR TITLE
Fix broken link for Contributors Guide

### PR DESCRIPTION
**Pull request by appleboy**
as title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 ### Important Notes
 
-1. **YOU MUST READ THE [Contributors Guide](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+1. **YOU MUST READ THE [Contributors Guide](./CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
```
Comment (tboerger) [inline README.md]: I think even the./ is optional :)

**Comment by tboerger** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (appleboy) [reply README.md]: Yes. :)

Comment (lunny) [reply README.md]: could remove `./`? :)

Comment (tboerger) [reply README.md]: I don't have a strong opinion on that. If you say lgtm feel free to merge it like it is

**Comment by appleboy** [comment]
Done. Please squash and merge.

**Comment by lunny** [comment]
LGTM

**Pull request merged**